README.md

```diff
@@ -122,6 +123,16 @@ export OPENAI_API_KEY=your_api_key
 python examples/simple_robot_agent.py --backend=openai
 ```
 
+Alternatively, you can use another Colab to see our **Cognitive agent** in action within a **SimplerEnv** simulation. 
```

Comment:
## Getting Started

Run an OpenAI client prompt-tuned to output X,Y,Z roll pitch yaw actions in the SimplerEnv simulation:


your stuff


To run an OpenAI client real robot hardware, you can run the following (see our [colab]() for an in-depth tutorial):
```shell
export OPENAI_API_KEY=your_api_key
python examples/simple_robot_agent.py --backend=openai
```